template/packages/frontend/pages/index.tsx

```diff
@@ -1,210 +1,71 @@
 import Head from "next/head"
 import Link from "next/link"
+import {
+  Container,
+  Title,
+  Description,
+  Card,
+  Code,
+  Main,
+  Grid,
+  Footer,
+  Logo,
+} from "../components/IndexStyles"
 
 export default function Home() {
   return (
-    <div className="container">
+    <Container>
       <Head>
         <title>Create a new Originate App</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main>
-        <h1 className="title">
+      <Main>
+        <Title>
           Welcome to{" "}
           <a href="https://github.com/Originate/create-react-app">
             Create Originate App!
           </a>
-        </h1>
+        </Title>
 
-        <p className="description">
-          Get started by editing <code>pages/index.js</code>
-        </p>
+        <Description>
+          Get started by editing <Code>pages/index.js</Code>
+        </Description>
 
-        <div className="grid">
-          <a href="https://nextjs.org/docs" className="card">
+        <Grid>
+          <Card href="https://nextjs.org/docs">
             <h3>Documentation &rarr;</h3>
             <p>Find in-depth information about Next.js features and API.</p>
-          </a>
+          </Card>
 
-          <a href="https://nextjs.org/learn" className="card">
+          <Card href="https://nextjs.org/learn">
             <h3>Learn &rarr;</h3>
             <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
+          </Card>
 
-          <a
-            href="https://github.com/vercel/next.js/tree/master/examples"
-            className="card"
-          >
+          <Card href="https://github.com/vercel/next.js/tree/master/examples">
             <h3>Examples &rarr;</h3>
             <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
+          </Card>
 
-          <Link href="/recipes">
-            <a className="card">
+          <Link href="/recipes" passHref={true}>
+            <Card>
               <h3>Try the Recipes Demo &rarr;</h3>
               <p>See a working example of Create Originate App. It's cool.</p>
-            </a>
+            </Card>
           </Link>
```

Comment:
When the child component is `<a>` then `<Link>` automatically sets the `href` prop on the child. In other cases we apparently need to set `passHref={true}` to make `<Link>` set `href` on the child. If we don't do this with styled links then the user's cursor will not properly change to a pointer when hovering the link.